### PR TITLE
Propage enableCodeAnalyzersOnTestApps setting to Run-AlPipeline

### PR DESCRIPTION
**Pull request by mazhelez**
Pending on https://github.com/microsoft/navcontainerhelper/pull/3292 and BcContainerHelper preview release.

### CODE REVIEW

File: Actions/AL-Go-Helper.ps1 [hunk 1]
```diff
@@ -18,7 +18,7 @@ $defaultCICDPushBranches = @( 'main', 'release/*', 'feature/*' )
 [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'defaultCICDPullRequestBranches', Justification = 'False positive.')]
 $defaultCICDPullRequestBranches = @( 'main' )
 $runningLocal = $local.IsPresent
-$defaultBcContainerHelperVersion = "preview" # Must be double quotes. Will be replaced by BcContainerHelperVersion if necessary in the deploy step
+$defaultBcContainerHelperVersion = "https://github.com/mazhelez/navcontainerhelper/zipball/run-cops-test-apps" # Must be double quotes. Will be replaced by BcContainerHelperVersion if necessary in the deploy step
```
Comment (mazhelez) [inline Actions/AL-Go-Helper.ps1:21]: Revert

```suggestion
$defaultBcContainerHelperVersion = "preview" # Must be double quotes. Will be replaced by BcContainerHelperVersion if necessary in the deploy step
```

**Comment by mazhelez** [comment]
Doc needs to be provided.

**Comment by mazhelez** [comment]
> Doc needs to be provided.

Added,

**Review by freddydk — APPROVED** [review]

**Comment by freddydk** [comment]
LGTM

**Review by elbardel — APPROVED** [review]

**Pull request merged**